kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-common/src/main/java/org/kie/server/remote/rest/common/resource/KieServerRestImpl.java

```diff
@@ -126,6 +138,24 @@ public Response listContainers(@Context HttpHeaders headers,
         return createCorrectVariant(server.listContainers(containerFilter), headers);
     }
 
+    @GET
+    @Path("prometheus")
+    @Produces({MediaType.TEXT_PLAIN})
+    public Response getModels() {
```

Comment:
why this endpoint is in KieServerRestImpl and not inside the Prometheus KieServer extension?

Reply:
Done